feature/identity/src/main/java/com/revenuecat/purchases/identity/IdentityManager.kt

```diff
@@ -41,6 +46,7 @@ class IdentityManager(
         deviceCache.cacheAppUserID(appUserIDToUse)
         subscriberAttributesCache.cleanUpSubscriberAttributeCache(appUserIDToUse)
         deviceCache.cleanupOldAttributionData()
+        invalidateCustomerInfoAndETagCacheIfNeeded(appUserIDToUse)
```

Comment:
I was thinking about this for a while, and I think invalidating it on first configure should cover all cases, since when we `logIn`, we need a new `CustomerInfo` and we also delete the old cached `CustomerInfo` for the previous user.

I was also thinking about where this logic belongs to, and I thought that it fit well within the `IdentityManager`. Lmk if you had other ideas though!

Reply:
I like this, all combined in a single place 👍🏻